syntax = "proto3";

message TodoCreate {
  int32 id= 1;
  string title= 2;
  string description = 3;
  bool completed= 4;
}



message TodoUpdate {
  int32 id= 1;
  string title= 2;
  string description = 3;
  bool completed= 4;
}



message Todo {
  int32 id = 1;
  string title = 2;
  string description = 3;
  bool completed = 4;
}

message TodoList {
  repeated Todo todos = 1;
}


message TodoDelete {
  int32 id = 1;
}